### PR TITLE
Multiple commits in the experiments table

### PR DESCRIPTION
**Pull request by sroy3**
Part of #1966 
Screen Shot 2022-09-13 at 2 23 18 PM
 Quite straight forward. Props to @mattseddon for thinking ahead, because this just works.

### CODE REVIEW

**Comment by sroy3** [comment]
https://user-images.githubusercontent.com/3683420/190235297-4c0c98d2-1832-42df-af44-66dfe774622c.mov

**Review by mattseddon — APPROVED** [review]
Initially, I was very confused by the chevron placement of the second commit but it does seem like everything is working as expected:

Screen Shot 2022-09-15 at 4 35 13 pm (2)

I do think we now need to nest experiments underneath their respective commit in the experiments tree:

Screen Shot 2022-09-15 at 4 38 51 pm (2)

The top of the tree is flat and fairly confusing. We should follow up and make that change ASAP.

There is at least one function (`collectWorkspaceRunningCheckpoint`) which is (probably) checking previously commits when it doesn't need to be.

We should also add a small multi-commit test fixture :pray:

**Comment by mattseddon** [comment]
Unable to apply an experiment from a previous commit: 

https://user-images.githubusercontent.com/37993418/190335647-3348ba64-3498-4b40-b70f-8844e5bb2f65.mov

This doesn't work for either the commit/short sha or the name. It looks like `exp apply` will only accept names from the current commit. That means we need to raise a bug in DVC and suppress the option from the tree/palette/table for previous commits (for the time being).

File: extension/src/cli/dvc/constants.ts [hunk 1]
```diff
@@ -28,9 +28,11 @@ export enum SubCommand {
 }
```
Comment (julieg18) [inline extension/src/cli/dvc/constants.ts:28]: Looks like the row borders are now a bit buggy which I believe is due to [this line in our styles](https://github.com/iterative/vscode-dvc/blob/84a9883203a063265cad7aab8d56220634d8498c/webview/src/experiments/components/table/styles.module.scss#L239):

Screen Shot 2022-09-15 at 7 43 29 AM

Comment (julieg18) [reply extension/src/cli/dvc/constants.ts:28]: If I remember correctly, the CSS lines were added to make the last row on the table not have a border 🤔

**Review by julieg18 — APPROVED** [review]
Great work! Agreed with Matt comments. I was also confused about the placement of the commit rows since they're at the same level as the `main` branch.

**Review by wolmir — APPROVED** [review]

**Pull request merged**